Source/UI/NewUI/BillingForm/ViewController/BillingFormViewController.swift

```diff
@@ -59,6 +59,9 @@ final class BillingFormViewController: UIViewController {
         view.backgroundColor = .clear
         view.register(CellTextField.self)
         view.register(CellButton.self)
+        //TODO: 1240 backgroundColor should be injected and allowed by merchants to be customized.
```

Comment:
As discussed @ehab-al-cko you can get rid of these todos once you fix the issue. all todos for //TODO: 1240